libkbfs/folder_branch_ops.go

```diff
@@ -3253,6 +3258,18 @@ func (fbo *folderBranchOps) applyMDUpdates(ctx context.Context,
 	return fbo.applyMDUpdatesLocked(ctx, lState, rmds)
 }
 
+func (fbo *folderBranchOps) getLatestRemoteMergedRevision() MetadataRevision {
```

Comment:
Actually scratch that.  For this one, since you only use it in a place where the lock isn't yet taken, you can just take an `lState`, and then `fbo.headLock.RLock(lState)` here.